main/src/cgeo/geocaching/location/GeopointFormatter.java

```diff
@@ -98,8 +98,8 @@ public static String format(final Format format, final Geopoint gp) {
 
             case LAT_LON_DECMINUTE_SHORT:
                 return String.format(Locale.getDefault(), "%c%d %02d%c%03d %c%d %02d%c%03d",
```

Comment:
```suggestion
                return String.format((Locale) null, "%c%d %02d.%03d %c%d %02d.%03d",
```
and remove the `'.'` arguments.

Reply:
@eddiemuc Also applies to new version

Reply:
done